test/test_ops.py

```diff
@@ -771,20 +777,37 @@ def test_variant_consistency_jit(self, device, dtype, op):
         include_conjugated_inputs = op.test_conjugated_samples and dtype.is_complex
         samples = op.sample_inputs(device, dtype, requires_grad=_requires_grad, include_conjugated_inputs=include_conjugated_inputs)
 
-        for sample in samples:
-            # Acquires variants to test
-            func = op.get_op()
-            method = op.get_method()
-            variants = {
-                # TODO: inplace tests currently fail, fix and add inplace variant
-                'function': func, 'method': method,
-            }
+        # Acquires variants to test
+        func = op.get_op()
+        method = op.get_method()
+        variants = {
+            # TODO: inplace tests currently fail, fix and add inplace variant
+            'function': func, 'method': method,
+        }
+
+        # TODO: find better way to standardize on op registration itself..
+        has_fake_function = op.name in ["resize_", 'resize_as_']
+
+        if has_fake_function:
```

Comment:
I would just add this attribute to the OpInfos for now

Reply:
fine doing that, for two functions it seems like a bit of overkill... but i dont know how prevalent it will be elsewhere